extensions/wikia/AdEngine/js/template/porvata.js

```diff
@@ -217,8 +213,8 @@ define('ext.wikia.adEngine.template.porvata', [
 
 		log(['show', params], log.levels.debug, logGroup);
 
-		if (prebid && params.hbAdId) {
-			params.bid = bidders && bidders.isEnabled() ? bidders.getBidByAdId(params.hbAdId) : prebid.getBidByAdId(params.hbAdId);
+		if (bidders && params.hbAdId) {
```

Comment:
Should we check whether `bidders` are enabled?

Reply:
Yes we can, fixed.